extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SchedulerRuntimeConfig.java

```diff
@@ -13,4 +15,9 @@
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
-}
\ No newline at end of file
+    /**
+     * Scheduled task will be flagged as overdue if next execution time is exceeded by this period.
+     */
+    @ConfigItem(name = "grace-period", defaultValue = "1S")
+    public Duration gracePeriod;
```

Comment:
Should this be `overdueGracePeriod`?

Reply:
In the label also or just the variable name ?

Reply:
I'd suggest that the user see `quarkus.scheduler.overdue-grace-period`.

If you change the variable name, the configuration name will be automatically generated from it.

Reply:
ok thanks, it's done !